test/test_target_plots.py

```diff
@@ -4,20 +4,23 @@
 import re
 import pandas as pd
 import numpy as np
-from numpy.testing import assert_array_equal
```

Comment:
This is all OK for now.  As I mentioned in the initial merge, this test file needs a re-work to be plot-oriented.  When that rework happens, we can make better decisions about deleting stuff.  But commenting out for now is fine.